docs/deprecated/designs/scheduled_capacity.md

```diff
@@ -1,8 +1,9 @@
 # Scheduled Capacity Design
+*Authors: njtran@*
```

Comment:
Since this was for an old version of Karpenter, I'm fine with removing this from the repo.